wgsl/index.bs

```diff
@@ -1619,6 +1627,65 @@ texel_format
 
 </pre>
 
+## Atomic Types ## {#atomic-types}
+
+<table class='data'>
+  <thead>
+    <tr><th>Type<th>Description
+  </thead>
+  <tr><td>atomic<*T*><td>Atomic of type *T*. *T* must be either [=u32=] or [=i32=].
+</table>
+
+Operations on atomic objects in [SHORTNAME] are mutually ordered for each
+object. That is, multiple invocations may modify the object without causing a
+data-race.
+
+Atomic objects may only be operated on by the [[#atomic-builtin-functions]].
+
+Atomic types may only be instantiated by `read_write` access variables in the
+[=storage classes/storage=] or [=storage classes/workgroup=] storage classes.
```

Comment:
Suggest moving this to before 1639, because that gives context to where atomic objects can exist.

Also, this writing makes it seem that read_write access applies to workgroup storage class. Suggest reordering like this instead:

> Atomic types may only be instantiated by variables in the
`read_write` access variables in the  [=storage classes/workgroup=] storage class,
or by `read_write` access variables in the [=storage classes/storage=] storage class.